tests/shardddl4_1/run.sh

```diff
@@ -696,21 +710,36 @@ function DM_151_CASE {
 	run_sql_source2 "insert into ${shardddl1}.${tb1} values(5,5);"
 	run_sql_source2 "insert into ${shardddl1}.${tb2} values(6,6);"
 
-	run_dm_ctl_with_retry $WORK_DIR "127.0.0.1:$MASTER_PORT" \
-		"query-status test" \
-		"Unsupported modify column: type double not match origin int(11), and tidb_enable_change_column_type is false" 1
+	if [[ "$shardmode" == "pessimistic" ]]; then
```

Comment:
If this case now is supported, we may change it to a positive case, and use check_sync_diff below.

Reply:
added